pkg/volume/util/nestedpendingoperations/nestedpendingoperations.go

```diff
@@ -47,8 +47,11 @@ const (
 type NestedPendingOperations interface {
 	// Run adds the concatenation of volumeName and podName to the list of
 	// running operations and spawns a new go routine to execute operationFunc.
-	// If an operation with the same volumeName and same or empty podName
-	// exists, an AlreadyExists or ExponentialBackoff error is returned.
+	// If an operation with the same volumeName, same or empty podName
+	// and same operationName exits, an AlreadyExists or ExponentialBackoff
```

Comment:
An AlreadyExists error is returned for the same volumeName and same podName, as long as the previous operation is pending, right?

Reply:
yeah, operations with same volumename and podName aren't allowed to proceed concurrently. If there is a pending operation with same volumenme+podName there `AlreadyExists` error is thrown.

Reply:
could you update the comments with that explanation?